index.js

```diff
@@ -37,40 +42,42 @@ const init = (fn, filePath, data, options) => {
 
 const main = async (filePath, data, options) => {
 	let {indent} = options;
-
-	if (options.detectIndent) {
-		try {
-			const file = await readFile(filePath, 'utf8');
+	let EOF = '\n';
```

Comment:
`EOF` is not the correct term for this.

Reply:
Why not? End-of-file? Doesn't it make sense in context?
```
`${json}${EOF}`
```

And if not, what do you prefer? `trailingNewline`?

Reply:
EOF is a special character: https://en.wikipedia.org/wiki/End-of-file

> trailingNewline?

Yes